### PR TITLE
New package: HAOPLAY.GirlsFrontline2 version 1.0.1

### PR DESCRIPTION
**Pull request by solomoncyj**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/195642)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 899.)

**Comment by solomoncyj** [comment]
i have redownloaded the exe, but the sha 256 is the same

File: manifests/h/HAOPLAY/GirlsFrontline2/1.0.1/HAOPLAY.GirlsFrontline2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: HAOPLAY.GirlsFrontline2
+PackageVersion: 1.0.1
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2024-12-02
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gf2-dl.haoplay.com/pc/GF2_v1.0.1.exe
+  InstallerSha256: 47D89B9AF7EAC939BD52AD33A59BDC30561580FFF2028EADFEEB6B393B3EF701
```
Comment (stephengillie) [inline manifests/h/HAOPLAY/GirlsFrontline2/1.0.1/HAOPLAY.GirlsFrontline2.installer.yaml:12]:
```suggestion
  InstallerSha256: 27A52AEE3584EAE3B282E5128BFFC0D4565BDCDCCA2FE6ABAB50A366FA980383
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> 12/3/2024 7:06:06 AM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   27A52AEE3584EAE3B282E5128BFFC0D4565BDCDCCA2FE6ABAB50A366FA980383
Expected hash: 47d89b9af7eac939bd52ad33a59bdc30561580fff2028eadfeeb6b393b3ef701

**Pull request closed without merging**